core/src/main/java/com/lionparcel/commonandroid/header/res/layout/lp_layout_header_ca.xml

```diff
@@ -103,6 +103,7 @@
         android:layout_height="wrap_content"
         android:layout_marginEnd="@dimen/spacing_xl"
         android:textAppearance="@style/Body3Semibold"
+        android:fontFamily="@font/poppins_semi_bold"
```

Comment:
Why not change the font in fonts.xml ?

Reply:
Tapi untuk case bold atau regular bisa soalnya ada `android:textStyle="bold/normal"`